apps/src/templates/SchoolInfoConfirmationDialog.jsx

```diff
@@ -21,24 +21,46 @@ class SchoolInfoConfirmationDialog extends Component {
 
   constructor(props) {
     super(props);
-
     this.state = {
       showSchoolInterstitial: false,
       schoolName: props.schoolName,
```

Comment:
Should this be required? (Github won't let me put this comment comment in propTypes)

Reply:
@bethanyaconnor Are you referring to schoolName?  There might be times that a teacher did not enter the name of the school; so I decided to have a default value of an empty string

Reply:
Makes sense. Thanks!